### DISCUSSION TITLE
Better Documentation for The Pathology App

### DISCUSSION THREAD

**Discussion by OmarAshkar** [Project-MONAI/MONAILabel#1062] [Q&A] [answered]
Hello,

I am very noob to pathology and deep learning, so I am asking if I can get clear documentation on the pathology app. 

I have histology images with multi-cells and glands, I want to label them to the finest level. Cells are both nucleus and cytoplasm stained. I tried to run DeepEdit and Nuclick from Qupath. 

From reading the documentation, it seems that DeepEdit and Nuclick can annotate 2 classes: positive and negative. To put that into a test, I have manually labeled 2 types of cells into positive and negative, submitted them to the server, selected a small region, and run a new annotation. Yet things do not work as expected:
1. The selected cells are not added to any class. Just highlighted cells.
2. I cannot modify the result of the annotation no matter if I selected only other totally different cells. It seems like the algorithm it hard coded and does not use manual annotation.

It would be better if there is better documentation for starters like me, but any guidance is highly appreciated.

Kind Regards!

**Comment by tangy5** [comment]
Thank you for this feedback. The suggestion is useful to the development. 
We will discuss improving the pathology documentation. Thanks.

**Comment by SachidanandAlle** [comment, marked as answer]
Classification is currently not supported yet.  We are looking for more contributions as well... otherwise the plan is to support classifying the cells in upcoming version 0.6 (possibly in Dec) for pathology

**Reply by OmarAshkar** [reply]
@SachidanandAlle  Thank you for the clarification. So, is there currently a way to override or update the default cells interactively using DeepEdit or NuClick? Even if I can get one kind of cell accurately for my image type?
Thank you !

**Reply by SachidanandAlle** [reply]
You can do that in the Client (qupath or dsa supports changing the annotation class).. but possibly saving label into monailabel won't help.. instead you can use save function from the qupath to save your annotation work

**Reply by diazandr3s** [reply]
Hi @OAshkar,

Here are two videos showing how you can run inference on currently available models: one for [QuPath + MONAI Label](https://drive.google.com/file/d/18mQ5DXuThp9YxXcbR0f19yS2klhmZozG/view?usp=sharing) and one for [DSA + MONAI Label](https://drive.google.com/file/d/19j7CMx7c8nn96ZL75L1a26usw8pHW-Sj/view?usp=sharing).

As @SachidanandAlle mentioned, segmentation + classification isn't available yet - only segmentation

Hope this helps,

**Reply by OmarAshkar** [reply]
Thank you @diazandr3s . Extermly helpful! Just to let is sync, there is no way to let DeepEdit use the manual annotation as a reference for segmentation?

**Reply by diazandr3s** [reply]
I'm glad to hear you found the videos useful.

> there is no way to let DeepEdit use the manual annotation as a reference for segmentation?

Yes, you can use the manually created segmentations to train DeepEdit. Once it is trained, DeepEdit can be used for automatic segmentation, and segmentation correction using clicks. Here is the article showing DeepEdit on radiology applications: https://link.springer.com/chapter/10.1007/978-3-031-17027-0_2

It also applies to pathology applications.

Hope this makes sense,
